### PR TITLE
tests: drivers: build_all: sensor: Adding Festo VEAA pressure regulator

### PR DESCRIPTION
**Pull request by soburi**
Add "festo,veaa-x-3" to testing targets.

### CODE REVIEW

File: tests/drivers/build_all/sensor/adc.dtsi [hunk 1]
```diff
@@ -116,3 +116,13 @@ test_seeed_grove_temperature: seeed-grove-temperature {
 	compatible = "seeed,grove-temperature";
 	io-channels = <&test_adc 0>;
 };
+
+test_veaa_x_3: test_veaa_x_3 {
+	status = "okay";
+	compatible = "festo,veaa-x-3";
+	io-channels = <&test_adc 3>;
+	dac = <&test_adc>;
```
Comment (jeppenodgaard) [inline tests/drivers/build_all/sensor/adc.dtsi:124]: `test_dac`?

Comment (soburi) [reply tests/drivers/build_all/sensor/adc.dtsi:124]: Oops!
I fixed it. Thank you.

Comment (ubieda) [reply tests/drivers/build_all/sensor/adc.dtsi:124]: Hmm - I still see `test_adc` instead of `test_dac`. Could you revisit this?

**Review by ubieda — COMMENTED** [review]
Looks good other than the unresolved review by @jeppenodgaard. I'll +1 once that's addressed.

**Review by MaureenHelm — APPROVED** [review]

**Comment by soburi** [comment]
@ubieda 
> Hmm - I still see test_adc instead of test_dac. Could you revisit this?

Sorry, I missed it in the commit. I'm sorry for the trouble, but please check it again.

**Review by ubieda — APPROVED** [review]

**Pull request merged**